### ISSUE TITLE
cannot read .sk2 files linux -> win

### ISSUE THREAD

**Issue by mhekeler** [sk1project/sk1-wx#172]
sk1 on windows cannot open files which was created on linux sk1. A dialog opens "Opening file - Parsing in Progress" and stays forever.

linux can open the files which werde created on windows.

**Comment by sk1project** [comment]
@mhekeler could you provide a sample file? We have tested this issue and cannot reproduce the problem.

**Comment by mhekeler** [comment]
I made some tests with multiple files and different content.
And I realized that all the testfiles which I create on linux version could be opener in win version.

So it must be a problem in my specific file. The main difference between the testfiles and the problematic file is that the problematic file has two pages. 
So I will make some additional tests with 2-page-files as soon as I have access to a windows box.

I am sorry that I can't attach the problematic file here because there are some jpg in the file containing some still unsettled copyright issues.
